react-frontend/src/components/DigitalFramework/digitalframework.js

```diff
@@ -0,0 +1,160 @@
+    'Modernize the way government funds technology.',
+    'Provide clear direction to government on the use of modern tools and infrastructure.',
+    'Create a program to develop and retain existing digital talent within the BC Public Service, and to attract new talent from outside government.',
+  ],
+  [
+    'Extend and share B.C.’s world-leading digital procurement programs, such as Sprint With Us.',
+    'Launch digital.gov.bc.ca, a website that will showcase our work on digital government.',
+    'Continue expanding connectivity for rural and Indigenous communities in B.C.',
+    'Provide guidance and support for developing data commons.',
+  ],
+];
+
+const objectives = [
+  [
+    'Online services meet British Columbians’ modern expectations.',
+    'People will have a common experience of government, and digital teams will be able to deliver services quickly and cost-effectively.',
+    'British Columbians will have a safe, common way to identify who they are when interacting with government.',
+    'Faster, more streamlined services for B.C. businesses.',
+    'Two new online services – fast, simple and intuitive to use – meet the needs of British Columbians.',
+  ],
+  [
+    'Better use of data to support the public good while protecting sensitive and personal information.',
+    'Co-develop a trusted approach to using digital technologies and data to better serve people and businesses in British Columbia.',
+    'Improve decision-making about some of B.C.’s most complex issues.',
+    'Improve service delivery for citizens, who will no longer need to understand how government is organized to find the services they need.',
+  ],
+  [
+    'BC Public Service employees will have the access, support and guidance they need to use modern tools and ways of working.',
+    'Funding for priority digital initiatives will be evidence-based, faster, more flexible and agile, and ensure better value for citizens.',
+    'Digital teams will have the tools they need to deliver better value to citizens, faster.',
+    'The BC Public Service will be better equipped to deliver government’s priorities.',
+  ],
+  [
+    'Government procurement will help grow local economies, create local jobs, and take advantage of made-in-B.C. innovations.',
+    'B.C.’s work on digital transformation will be shared and amplified, helping us connect with partners.',
+    'Increased economic and social development, educational opportunities, access to online services, health and public safety for all British Columbians.',
+    'Enhanced collaboration with industry and other levels of government and First Nations to solve complex problems using data.',
+  ],
+];
+
+function DigitalFramework() {
+  return (
+    <div className="digitalContainer">
+      <DigitalHeader />
+      <div className="digitalBody">
+        <DigitalBlock title={titles[0]} paragraph={descriptions[0]} />
```

Comment:
sounds like this could be rendered in a loop

Reply:
Considered that but there are blocks which are not exactly the same (containing lists and links) so it would require conditionals in the child component which might get ugly